tests/unit/Utils/DocLinkTest.php

```diff
@@ -11,7 +11,7 @@
  */
 class DocLinkTest extends \PHPUnit\Framework\TestCase {
 	public function testGetServerUrl() {
-		$expected = 'https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
+		$expected = 'https://doc.owncloud.com/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
```

Comment:
Is this link expected to be working?

Reply:
it is fixed. please review again @pako81